cli/src/complete.rs

```diff
@@ -159,6 +159,32 @@ pub fn aliases() -> Vec<CompletionCandidate> {
     })
 }
 
+pub fn operations() -> Vec<CompletionCandidate> {
+    with_jj(|mut jj, _| {
+        let output = jj
+            .arg("operation")
+            .arg("log")
+            .arg("--no-graph")
+            .arg("--limit")
+            .arg("1000")
```

Comment:
I set the limit to 100, let me know if you think something else is better.

I also added a `display_order` to the candidates, they should now be ordered the same way jj prints them.

Reply:
> I set the limit to 100, let me know if you think something else is better.

Looks good, thanks.

> I also added a display_order to the candidates, they should now be ordered the same way jj prints them.

Unfortunately, no. It might be an issue specific to zsh or my zshrc. Do you see any difference in your shell? If the outputs are identical, remove `display_order`.

Reply:
Alright, I removed the `display_order`. I didn't see a difference in fish either. In fish, the completions are apparently always ordered the same way they are emitted by us, so I only used `display_order` when I wanted to deviate from that.